news-bundle/src/Resources/contao/dca/tl_layout.php

```diff
@@ -22,7 +22,7 @@
 (
 	'exclude'         => true,
 	'inputType'       => 'checkbox',
-	'foreignKey'      => 'tl_news_feed.title',
```

Comment:
thinking about this, this should probably be a page picker instead? A checkbox list of pages does not make much sense 😆

Reply:
It is limited to only feed type pages in the corresponding listener, so it does not display so many pages.